src/storage/src/table/cell_based_table.rs

```diff
@@ -332,6 +335,100 @@ impl<S: StateStore> CellBasedTable<S> {
         .await
     }
 
+    fn serialized_pk_bound(
+        &self,
+        pk_prefix: &Row,
+        next_col_bound: Bound<&Datum>,
+        is_start_bound: bool,
+    ) -> StorageResult<Bound<Vec<u8>>> {
```

Comment:
So it seems unnecessary to return `StorageResult<...>`?

Reply:
serialize_pk returns Result

Reply:
I think `serialize_pk` should not return `StorageResult` as well...

Reply:
Yes it returns Ok 😇